packages/angular/projects/ui-angular/src/lib/components/authenticator/components/reset-password/reset-password.component.ts

```diff
@@ -1,21 +1,28 @@
-import { Component, HostBinding, Input } from '@angular/core';
+import { Component, HostBinding, Input, OnInit } from '@angular/core';
 import { AuthenticatorService } from '../../../../services/authenticator.service';
-import { translate } from '@aws-amplify/ui';
+import { getAliasInfoFromContext, translate } from '@aws-amplify/ui';
 
 @Component({
   selector: 'amplify-reset-password',
   templateUrl: './reset-password.component.html',
 })
-export class ResetPasswordComponent {
+export class ResetPasswordComponent implements OnInit {
   @HostBinding('attr.data-amplify-authenticator-resetPassword') dataAttr = '';
   @Input() public headerText = translate('Reset your password');
 
   // translated texts
   public sendCodeText = translate('Send Code');
   public backToSignInText = translate('Back to Sign In');
+  public labelText = translate<string>('Username');
 
   constructor(public authenticator: AuthenticatorService) {}
 
+  ngOnInit(): void {
+    const { authState } = this.authenticator;
+    const { label } = getAliasInfoFromContext(authState.context);
+    this.labelText = `Enter your ${label.toLowerCase()}`;
```

Comment:
Just double checking, should the labelText be just 

```
"Username" | "Email" | "Phone"
```

or 

```ts
`Enter your {$label.toLowerCase)`
```?

I believe the original issue was that the label showed "username". My worry with this approach is that "`Enter your ${label.toLowerCase()}`" would be inconsistent with other auth field labels we have.

Reply:
This seems to be an inconsistency between Vue, React and Angular. React and Vue had 'Enter your username', while Angular had just 'Username'. So I fixed Angular so it matched Vue and React.

Reply:
I think the slight inconsistency between the different fields is fine. I like 'Enter your...' text.

Reply:
Ah got it! In that case I'm good.